src/AzureMySqlAction.ts

```diff
@@ -17,9 +17,26 @@ export default class AzureMySqlAction {
 
     public async execute() {
         core.debug('Begin executing action...');
-        
+
         let mySqlClientPath = await AzureMySqlActionHelper.getMySqlClientPath();
-        await exec.exec(`"${mySqlClientPath}" -h ${this._inputs.serverName} -D ${this._inputs.connectionString.database} -u ${this._inputs.connectionString.userId} ${this._inputs.additionalArguments} -e "source ${this._inputs.sqlFile}"`, [`--password=${this._inputs.connectionString.password}`]);
+
+        let parameters = [
+            "-h", this._inputs.serverName,
+            "-u", this._inputs.connectionString.userId,
+            "-e", `source ${this._inputs.sqlFile}`,
+        ]
+
+        if (this._inputs.connectionString.database) {
+            parameters.push("-D")
+            parameters.push(this._inputs.connectionString.database)
```

Comment:
this does not fix the issue when database name has spaces in it. Can we use backticks around database name so that spaces in db name issue is covered?

Reply:
Spaces are covered since parameters have been moved (except for `additionalArguments`) to `exec` args parameter which [handles quoting for us](https://github.com/actions/toolkit/blob/af821474235d3c5e1f49cee7c6cf636abb0874c4/packages/exec/src/exec.ts#L12)